packages/common/src/error.rs

```diff
@@ -49,6 +58,9 @@ pub enum ContractError {
     #[error("InvalidAddress")]
     InvalidAddress {},
 
+    #[error("FunctionDeclinesFunds")]
+    FunctionDeclinesFunds {},
```

Comment:
This is a useful error, we should definitely use this in other areas

Reply:
Another option could be using `cw_utils::nonpayable`